nginx-controller/controller/controller.go

```diff
@@ -609,12 +659,31 @@ func (lbc *LoadBalancerController) syncCfgm(task Task) {
 		}
 	}
 
+	mergeableIngresses := make(map[string]nginx.MergeableIngresses)
 	var ingExes []*nginx.IngressEx
 	ings, _ := lbc.ingLister.List()
 	for i := range ings.Items {
 		if !lbc.isNginxIngress(&ings.Items[i]) {
 			continue
 		}
+		if ings.Items[i].Annotations["nginx.org/mergible-ingress-type"] == "minion" {
+			master, err := lbc.findMasterForMinion(ings.Items[i])
+			if err != nil {
+				glog.Errorf("Ignoring Ingress %v(Minion): %v", ings.Items[i], err)
+				continue
+			}
```

Comment:
let's run this check `lbc.cnf.HasIngress(&ings.Items[i])` on the master

Reply:
Done.